client/gutenberg/extensions/map-block/add-point/index.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Wordpress dependencies
+ */
+
+import { __ } from '@wordpress/i18n';
+
+import { Component } from '@wordpress/element';
+
+import {
+	Button,
+	Dashicon,
+	Popover
+} from '@wordpress/components';
+
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+
+import LocationSearch from '../location-search';
+
+import './style.scss';
+
+export class AddPoint extends Component {
+
+	constructor() {
+		super( ...arguments );
+		this.onAddPoint = this.onAddPoint.bind( this );
+	}
+
+	onAddPoint( point ) {
+
+		this.props.onAddPoint( point );
```

Comment:
In cases like this, we could directly call `this.props.onAddPoint` without the need to define a wrapper method.

But also, this method doesn't seem to be used at this time, so we can directly remove it.

Reply:
Resolved in https://github.com/Automattic/wp-calypso/pull/27836/commits/92aa832fd43ffac9ffdbc63ec43c68a9d9346760